pages/7_📋_プレイリスト履歴.py

```
@@ -139,27 +145,135 @@ def format_created_at(value: str, tz_name: str) -> str:
     )
     st.dataframe(header_df, use_container_width=True, hide_index=True)
 
+    # 削除ボタン（作成者のみ表示）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)